packages/interactivity/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 -   Break up init with yielding to main to prevent long task from hydration. ([#58227](https://github.com/WordPress/gutenberg/pull/58227))
 
+### Bug Fixes
+
+-   Avoid initializing private stores as public when they have initial state. ([#58722](https://github.com/WordPress/gutenberg/pull/58722))
```

Comment:
The PR title is different than the title here.

Reply:
You are right; it's confusing. I'm going to cherry-pick those commits and create another PR just for the bug fix.

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/58754.